client/src/components/Header.tsx

```diff
@@ -25,12 +26,18 @@ const Header: React.FC<HeaderProps> = (props) => {
       padding={3}
       {...props}
     >
-      <Flex align="center" mr={5}>
+      <Flex align="center" mr={5} color = "blue">
         <Heading size="lg" letterSpacing={"tighter"}>
-          <Link href="/">cs148-helloworld</Link>
+          <Link href="/">T10-AnimeList</Link>
         </Heading>
       </Flex>
 
+      <div>

```

Comment:
I think we can remove the surrounding `<div> </div>` tags and just have `<Head>...</Head>`